bots/github/test-github

```diff
@@ -117,5 +117,28 @@ class TestGitHub(unittest.TestCase):
         count = api.get("/count")
         self.assertEqual(count, 1)
 
+class TestWhitelist(unittest.TestCase):
+    def setUp(self):
+        self.temp = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.temp)
+
+    def testLoad(self):
+        filename = os.path.join(self.temp, "whitelist")
+        with open(filename, "w") as f:
+            f.write("   one   \n   two\nthree\n\n")
+        whitelist = github.whitelist(filename)
+        self.assertIn("one", whitelist)
+        self.assertIn("two", whitelist)
+        self.assertIn("three", whitelist)
+        self.assertNotIn("four", whitelist)
+        self.assertNotIn("", whitelist)
+
+    def testNotPresent(self):
+        filename = os.path.join(self.temp, "whitelist")
+        whitelist = github.whitelist(filename)
+        self.assertEqual(len(whitelist), 0)
+
```

Comment:
I'd add a new test that calls `github.whitelist()` with the default argument, and assert that the length is > 0. That will ensure that reading the default whitelist does at least something sensible, the whitelist is at the right place, and it doesn't crash.